src/platform/user/profile/vet360/util/local-vet360.js

```diff
@@ -102,7 +102,7 @@ export const mockContactInformation = {
     addressLine2: '',
     addressLine3: '',
     addressPou: 'RESIDENCE/CHOICE',
-    addressType: 'MILITARY OVERSEAS',
+    addressType: 'OVERSEAS MILITARY',
```

Comment:
this now matches reality: https://github.com/department-of-veterans-affairs/vets-website/blob/0f539058825668a08f9518f56c85dec9dd96d852/src/platform/user/profile/vet360/constants/index.js#L12-L16